packages/core-flows/src/inventory/steps/validate-inventory-items-for-create.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ContainerRegistrationKeys,
+  MedusaError,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+import { InventoryNext } from "@medusajs/types"
+
+export const validateInventoryItemsForCreateStepId =
+  "validate-inventory-items-for-create-step"
+export const validateInventoryItemsForCreate = createStep(
+  validateInventoryItemsForCreateStepId,
+  async (
+    input: {
+      inventoryItem: InventoryNext.CreateInventoryItemInput
+      tag?: string
```

Comment:
suggestion: tag seems weird no? What is the tag when you consume the step, shouldn't we ask for variant_id?

Reply:
we could, I've kept to the terminology from api-v1, figured we kept it as tag instead of variant=_id to keep it abstract in case of overrides: https://github.com/medusajs/medusa/pull/6693#discussion_r1523708129